liquibase-standard/src/main/java/liquibase/datatype/DataTypeFactory.java

```diff
@@ -179,11 +176,14 @@ public LiquibaseDataType fromDescription(String dataTypeDefinition, Database dat
                 additionalInfo = splitTypeName[1];
             }
         }
+        if (dataTypeName.toLowerCase(Locale.US).equals("timestamptz")) {
```

Comment:
Hi @asashour,

Overall, changes look good to me, but I was wondering whether we should be modifying this `IF` block, because if I'm not wrong `timestamptz` column type is `Postgres` specific, but I guess if we do that it would imply a little refactoring on the `getTimeZoneAdditionInformation()`. What do you think?

cc: @filipelautert

Reply:
Thanks for the feedback.

It seems that `timestamptz` is relevant to PostgreSQL (not to Oracle for instance), so yes, let's do it for only it for now.

However, it may be worth mentioning that the original issue happens with also Oracle, but it is due to the fact that [this line](https://github.com/liquibase/liquibase/blob/master/liquibase-standard/src/main/java/liquibase/datatype/DataTypeFactory.java#L124) changes `TIMESTAMP(6) WITH TIME ZONE(13)` to `TIMESTAMP`, thus removing the `time zone`. This should be also fixed, possibly in another PR.

And this also adds the fact that possibly other databases needs to be tested as well.

Reply:
@asashour, one other option is if you want to merge the code changes made by you here with the ones I included in #6929, or I can do that and include both here.

Thanks,
Daniel.

Reply:
Thanks, I think we shouldn't handle both PRs at the same time. Once the other PR is merged, I would resolve the conflict, let's keep this PR for PostgreSQL, and the other one for Oracle.